.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: elixir
+elixir:
+  - 1.4.0
+otp_release:
+  - 19.0
+addons:
+  postgresql: '9.4'
+services:
+  - postgresql
+before_install:
+  - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16"
+  - export CHROMEDRIVER_BINARY=/usr/bin/chromium-browser
+  - export CHROMEDRIVER_ARGS=--no-sandbox
+  - /usr/bin/chromium-browser --version
```

Comment:
Actually not sure we need any of these lines now, originally was copying off someone elses travis.yml who used chromedriver but I think with phantomjs below all this might be unneeded? unsure about xvfb but shoudn't need chromium stuff anyway